README.md

```diff
@@ -382,6 +382,7 @@ use for your tables:
   columnar data
 - `MARKDOWN` - A style that follows Markdown syntax
 - `ORGMODE` - A table style that fits [Org mode](https://orgmode.org/) syntax
+- `FANCY` - A style that uses continuous border lines for a fancier display on terminal
```

Comment:
Can we call it `DOUBLE` or `DOUBLE_LINE` or `DOUBLE_BORDER` or something?

Reply:
Done - changed to `DOUBLE_BORDER`